diff --git a/GUI/addFlagDialog.m b/GUI/addFlagDialog.m
index 281986b..21466d2 100644
--- a/GUI/addFlagDialog.m
+++ b/GUI/addFlagDialog.m
@@ -139,8 +139,8 @@ function [returnVars, flagVal, commentText] = addFlagDialog( variable, kVar, def
   set(optList,       'Position', [0.0,  0.9,  1.0,  0.1]);
   set(setPanel,      'Position', [0.0,  0.5,  1.0,  0.4]);
   set(commentPanel,  'Position', [0.0,  0.1,  1.0,  0.4]);
-  set(cancelButton,  'Position', [0.0,  0.0,  0.5,  0.1]);
-  set(confirmButton, 'Position', [0.5,  0.0,  0.5,  0.1]);
+  set(cancelButton,  'Position', [0.6,  0.015,  0.15,  0.07]);
+  set(confirmButton, 'Position', [0.8,  0.015,  0.15,  0.07]);
 
   % populate the panel for variable selection
   uicontrol(...
diff --git a/GUI/dataFileStatusDialog.m b/GUI/dataFileStatusDialog.m
index 40a8cd5..68e8cd4 100644
--- a/GUI/dataFileStatusDialog.m
+++ b/GUI/dataFileStatusDialog.m
@@ -116,13 +116,13 @@ function [deployments files] = dataFileStatusDialog( deployments, files )
 
   % position widgets
   set(f,                'Position', [0.3, 0.3,  0.4, 0.4 ]);
-  set(cancelButton,     'Position', [0.0, 0.0,  0.5, 0.08]);
-  set(confirmButton,    'Position', [0.5, 0.0,  0.5, 0.08]);
+  set(cancelButton,     'Position', [0.6, 0.015,  0.15, 0.05]);
+  set(confirmButton,    'Position', [0.8, 0.015,  0.15, 0.05]);
   set(fileList,         'Position', [0.0, 0.08, 1.0, 0.32]);
-  set(fileRemoveButton, 'Position', [0.0, 0.40, 0.5, 0.08]);
-  set(fileAddButton,    'Position', [0.5, 0.40, 0.5, 0.08]);
+  set(fileRemoveButton, 'Position', [0.4, 0.415, 0.2, 0.05]);
+  set(fileAddButton,    'Position', [0.7, 0.415, 0.25, 0.05]);
   set(depList,          'Position', [0.0, 0.48, 1.0, 0.44]);
-  set(depRemoveButton,  'Position', [0.0, 0.92, 1.0, 0.08]);
+  set(depRemoveButton,  'Position', [0.5, 0.93, 0.3, 0.05]);
   
   % reset back to pixel units
   set(f,                                 'Units', 'pixels');
diff --git a/GUI/datePromptDialog.m b/GUI/datePromptDialog.m
index c7b6ecc..cb4df00 100644
--- a/GUI/datePromptDialog.m
+++ b/GUI/datePromptDialog.m
@@ -109,8 +109,8 @@ function [year month day] = datePromptDialog(year,month,day)
   
   % position figure and widgets
   set(f,             'Position', [0.4,  0.46, 0.2,  0.08]);
-  set(cancelButton,  'Position', [0.0,  0.0,  0.5,  0.5 ]);
-  set(confirmButton, 'Position', [0.5,  0.0,  0.5,  0.5 ]);
+  set(cancelButton,  'Position', [0.6,  0.015,  0.15,  0.07 ]);
+  set(confirmButton, 'Position', [0.8,  0.015,  0.15,  0.07 ]);
   set(yearMenu,      'Position', [0.0,  0.5,  0.33, 0.5 ]);
   set(monthMenu,     'Position', [0.33, 0.5,  0.34, 0.5 ]);
   set(dayMenu,       'Position', [0.67, 0.5,  0.33, 0.5 ]);
diff --git a/GUI/exportDialog.m b/GUI/exportDialog.m
index 6a3a8ce..ab405e3 100644
--- a/GUI/exportDialog.m
+++ b/GUI/exportDialog.m
@@ -140,8 +140,8 @@ function [exportDir sets] = exportDialog( ...
   
   % position widgets
   set(f,             'Position', [0.25, 0.35, 0.5,  0.3]);
-  set(cancelButton,  'Position', [0.0,  0.0,  0.5,  0.1]);
-  set(confirmButton, 'Position', [0.5,  0.0,  0.5,  0.1]);
+  set(cancelButton,  'Position', [0.6,  0.015,  0.15,  0.07]);
+  set(confirmButton, 'Position', [0.8,  0.015,  0.15,  0.07]);
   set(dirLabel,      'Position', [0.0,  0.1,  0.15, 0.1]);
   set(dirText,       'Position', [0.15, 0.1,  0.8,  0.1]);
   set(dirButton,     'Position', [0.85, 0.1,  0.15, 0.1]);
diff --git a/GUI/listSelectionDialog.m b/GUI/listSelectionDialog.m
index c07516f..ea9602c 100644
--- a/GUI/listSelectionDialog.m
+++ b/GUI/listSelectionDialog.m
@@ -122,14 +122,14 @@ function [selected, cancel] = listSelectionDialog( ...
   
   % position the figure and widgets
   set(f,             'Position', [0.3, 0.36, 0.4, 0.28]);
-  set(cancelButton,  'Position', [0.0, 0.0,  0.5, 0.1]);
-  set(confirmButton, 'Position', [0.5, 0.0,  0.5, 0.1]);
+  set(cancelButton,  'Position', [0.6, 0.015,  0.15, 0.07]);
+  set(confirmButton, 'Position', [0.8, 0.015,  0.15, 0.07]);
   set(nSelList,      'Position', [0.0, 0.1,  0.4, 0.9]);
   set(selList,       'Position', [0.5, 0.1,  0.4, 0.9]);
-  set(downButton,    'Position', [0.9, 0.45, 0.1, 0.1]);
-  set(upButton,      'Position', [0.9, 0.55, 0.1, 0.1]);
-  set(remButton,     'Position', [0.4, 0.45, 0.1, 0.1]);
-  set(addButton,     'Position', [0.4, 0.55, 0.1, 0.1]);
+  set(downButton,    'Position', [0.91, 0.45, 0.08, 0.08]);
+  set(upButton,      'Position', [0.91, 0.55, 0.08, 0.08]);
+  set(remButton,     'Position', [0.41, 0.45, 0.08, 0.08]);
+  set(addButton,     'Position', [0.41, 0.55, 0.08, 0.08]);
   
   % reset back to pixels
   set(f,             'Units', 'pixels');
diff --git a/GUI/optionDialog.m b/GUI/optionDialog.m
index 4a998cc..6526541 100644
--- a/GUI/optionDialog.m
+++ b/GUI/optionDialog.m
@@ -95,8 +95,8 @@ function choice = optionDialog( title, message, options, default )
   set(confirmButton, 'Units', 'normalized');
 
   set(f,             'Position', [0.3,  0.46, 0.4,  0.08]);
-  set(cancelButton,  'Position', [0.0,  0.0,  0.5,  0.3 ]);
-  set(confirmButton, 'Position', [0.5,  0.0,  0.5,  0.3 ]);
+  set(cancelButton,  'Position', [0.6,  0.015,  0.15,  0.07 ]);
+  set(confirmButton, 'Position', [0.8,  0.015,  0.15,  0.07 ]);
   set(optList,       'Position', [0.0,  0.3,  1.0,  0.3 ]);
   set(msgPanel,      'Position', [0.0,  0.6,  1.0,  0.4 ]);
 
diff --git a/GUI/propertyDialog.m b/GUI/propertyDialog.m
index d8a55d2..547c698 100644
--- a/GUI/propertyDialog.m
+++ b/GUI/propertyDialog.m
@@ -111,8 +111,8 @@ function propertyDialog( filename, delim )
   
   % position dialog and widgets
   set(f,             'Position', [0.3, 0.3, 0.4, 0.4]);
-  set(cancelButton,  'Position', [0.0, 0.0, 0.5, 0.1]);
-  set(confirmButton, 'Position', [0.5, 0.0, 0.5, 0.1]);
+  set(cancelButton,  'Position', [0.6, 0.015, 0.15, 0.07]);
+  set(confirmButton, 'Position', [0.8, 0.015, 0.15, 0.07]);
   
   rlen = 0.9 / k;
   
diff --git a/GUI/siteDialog.m b/GUI/siteDialog.m
index e4bc945..fd229db 100644
--- a/GUI/siteDialog.m
+++ b/GUI/siteDialog.m
@@ -90,8 +90,8 @@ function selectedSiteId = siteDialog(siteId, siteDesc)
 
   % position the widgets
   set(f,               'Position', [0.2,  0.35,  0.6,   0.3]);
-  set(cancelButton,    'Position', [0.0,  0.0,   0.5,   0.1]);
-  set(confirmButton,   'Position', [0.5,  0.0,   0.5,   0.1]);
+  set(cancelButton,    'Position', [0.6,  0.015,   0.15,   0.07]);
+  set(confirmButton,   'Position', [0.8,  0.015,   0.15,   0.07]);
   set(fidLabel,        'Position', [0.0,  0.1,   0.2,   0.9]);
   set(fidList,         'Position', [0.2,  0.1,   0.8,   0.9]);
   
@@ -156,4 +156,4 @@ function selectedSiteId = siteDialog(siteId, siteDesc)
   % if user cancelled, return empty matrices
   if isempty(selectedSiteId), return; end
   
-end
\ No newline at end of file
+end
diff --git a/GUI/startDialog.m b/GUI/startDialog.m
index 1523735..b07cbd7 100644
--- a/GUI/startDialog.m
+++ b/GUI/startDialog.m
@@ -165,18 +165,18 @@ function [fieldTrip dataDir] = startDialog(mode)
   % position the widgets
   set(f,               'Position', [0.2,  0.35,  0.6,   0.3]);
 
-  set(cancelButton,    'Position', [0.0,  0.0,  0.5,   0.1]);
-  set(confirmButton,   'Position', [0.5,  0.0,  0.5,   0.1]);
+  set(cancelButton,    'Position', [0.6,  0.015,  0.15,   0.07]);
+  set(confirmButton,   'Position', [0.8,  0.015,  0.15,   0.07]);
 
   set(dirLabel,        'Position', [0.0,  0.1, 0.199, 0.1]);
-  set(dirText,         'Position', [0.2,  0.1, 0.65,  0.1]);
-  set(dirButton,       'Position', [0.85, 0.1, 0.15,  0.1]);
+  set(dirText,         'Position', [0.21,  0.1, 0.65,  0.08]);
+  set(dirButton,       'Position', [0.86, 0.1, 0.1,  0.08]);
 
   set(fidLabel,        'Position', [0.0,  0.2, 0.199, 0.7]);
   set(fidList,         'Position', [0.2,  0.2, 0.8,   0.7]);
   
-  set(dateStartButton, 'Position', [0.0,  0.9, 0.50,  0.1]);
-  set(dateEndButton,   'Position', [0.5,  0.9, 0.50,  0.1]);
+  set(dateStartButton, 'Position', [0.03,  0.91, 0.45,  0.08]);
+  set(dateEndButton,   'Position', [0.53,  0.91, 0.45,  0.08]);
   
   % reset back to pixels
   set(f,                                'Units', 'pixels');
diff --git a/NetCDF/template/global_attributes_timeSeries.txt b/NetCDF/template/global_attributes_timeSeries.txt
index 18280d1..a13c319 100644
--- a/NetCDF/template/global_attributes_timeSeries.txt
+++ b/NetCDF/template/global_attributes_timeSeries.txt
@@ -41,8 +41,8 @@ N, geospatial_lat_max       = [ddb Site Sites Latitude]
 N, geospatial_lon_min       = [ddb Site Sites Longitude]
 N, geospatial_lon_max       = [ddb Site Sites Longitude]
 N, instrument_nominal_height= 
-N, instrument_nominal_depth = 
-N, site_nominal_depth       = 
+N, instrument_nominal_depth = [ddb InstrumentDepth]
+N, site_nominal_depth       = [ddb Site Sites Depth]
 N, site_depth_at_deployment = 
 
 % if left empty, these fields are automatically populated upon import 
diff --git a/Preprocessing/pressureRelPP.m b/Preprocessing/pressureRelPP.m
index ac13d6e..e3b918f 100644
--- a/Preprocessing/pressureRelPP.m
+++ b/Preprocessing/pressureRelPP.m
@@ -138,8 +138,8 @@ if ~auto
     set(offsetFields,   'Units', 'normalized');
     
     set(f,             'Position', [0.2 0.35 0.6 0.3]);
-    set(cancelButton,  'Position', [0.0 0.0  0.5 0.1]);
-    set(confirmButton, 'Position', [0.5 0.0  0.5 0.1]);
+    set(cancelButton,  'Position', [0.6 0.015  0.15 0.07]);
+    set(confirmButton, 'Position', [0.8 0.015  0.15 0.07]);
     
     rowHeight = 0.9 / lenSam;
     for k = 1:lenSam
@@ -291,4 +291,4 @@ end
             offsets(idx) = val;
         end
     end
-end
\ No newline at end of file
+end
diff --git a/Preprocessing/timeMetaOffsetPP.m b/Preprocessing/timeMetaOffsetPP.m
index 3807360..b6cbcd9 100644
--- a/Preprocessing/timeMetaOffsetPP.m
+++ b/Preprocessing/timeMetaOffsetPP.m
@@ -137,8 +137,8 @@ function sample_data = timeMetaOffsetPP(sample_data, qcLevel, auto)
       set(offsetFields,   'Units', 'normalized');
       
       set(f,             'Position', [0.2 0.35 0.6 0.3]);
-      set(cancelButton,  'Position', [0.0 0.0  0.5 0.1]);
-      set(confirmButton, 'Position', [0.5 0.0  0.5 0.1]);
+      set(cancelButton,  'Position', [0.6 0.015  0.15 0.07]);
+      set(confirmButton, 'Position', [0.8 0.015  0.15 0.07]);
       
       rowHeight = 0.9 / length(sample_data);
       for k = 1:length(sample_data)
diff --git a/Preprocessing/timeOffsetPP.m b/Preprocessing/timeOffsetPP.m
index bdde13e..e6deb89 100644
--- a/Preprocessing/timeOffsetPP.m
+++ b/Preprocessing/timeOffsetPP.m
@@ -144,8 +144,8 @@ function sample_data = timeOffsetPP(sample_data, qcLevel, auto)
       set(offsetFields,   'Units', 'normalized');
       
       set(f,             'Position', [0.2 0.35 0.6 0.3]);
-      set(cancelButton,  'Position', [0.0 0.0  0.5 0.1]);
-      set(confirmButton, 'Position', [0.5 0.0  0.5 0.1]);
+      set(cancelButton,  'Position', [0.6 0.015  0.15 0.07]);
+      set(confirmButton, 'Position', [0.8 0.015  0.15 0.07]);
       
       rowHeight = 0.9 / length(sample_data);
       for k = 1:length(sample_data)
diff --git a/Preprocessing/timeStartPP.m b/Preprocessing/timeStartPP.m
index 91f3401..eaffaee 100644
--- a/Preprocessing/timeStartPP.m
+++ b/Preprocessing/timeStartPP.m
@@ -111,8 +111,8 @@ function sample_data = timeStartPP( sample_data, qcLevel, auto )
   set(startTimeFields, 'Units', 'normalized');
   
   set(f,             'Position', [0.2 0.35 0.6 0.3]);
-  set(cancelButton,  'Position', [0.0 0.0  0.5 0.1]);
-  set(confirmButton, 'Position', [0.5 0.0  0.5 0.1]);
+  set(cancelButton,  'Position', [0.6 0.015  0.15 0.07]);
+  set(confirmButton, 'Position', [0.8 0.015  0.15 0.07]);
   
   rowHeight = 0.9 / length(sample_data);
   for k = 1:length(sample_data)
diff --git a/Preprocessing/transformPP.m b/Preprocessing/transformPP.m
index dc95b02..67b4b72 100644
--- a/Preprocessing/transformPP.m
+++ b/Preprocessing/transformPP.m
@@ -124,8 +124,8 @@ function sample_data = transformPP( sample_data, qcLevel, auto )
       
       % position widgets
       set(f,             'Position', [0.25, 0.25, 0.5,  0.5]);
-      set(cancelButton,  'Position', [0.0,  0.0,  0.5,  0.1]);
-      set(confirmButton, 'Position', [0.5,  0.0,  0.5,  0.1]);
+      set(cancelButton,  'Position', [0.6,  0.015,  0.15,  0.07]);
+      set(confirmButton, 'Position', [0.8,  0.015,  0.15,  0.07]);
       set(tabPanel,      'Position', [0.0,  0.1,  1.0,  0.9]);
       
       % reset back to pixels
diff --git a/Preprocessing/variableOffsetPP.m b/Preprocessing/variableOffsetPP.m
index 8cbb836..af841ff 100644
--- a/Preprocessing/variableOffsetPP.m
+++ b/Preprocessing/variableOffsetPP.m
@@ -107,8 +107,8 @@ function sample_data = variableOffsetPP( sample_data, qcLevel, auto )
       
       % position widgets
       set(f,             'Position', [0.25, 0.25,  0.5, 0.5]);
-      set(cancelButton,  'Position', [0.0,  0.0,  0.5,  0.1]);
-      set(confirmButton, 'Position', [0.5,  0.0,  0.5,  0.1]);
+      set(cancelButton,  'Position', [0.6,  0.015,  0.15,  0.07]);
+      set(confirmButton, 'Position', [0.8,  0.015,  0.15,  0.07]);
       set(tabPanel,      'Position', [0.0,  0.1,  1.0,  0.9]);
       
       % reset back to pixels
